packages/dev/core/src/Engines/Extensions/engine.renderTarget.ts

```diff
@@ -109,7 +102,7 @@ ThinEngine.prototype.createRenderTargetTexture = function (this: ThinEngine, siz
     return rtWrapper;
 };
 
-ThinEngine.prototype._createDepthStencilTexture = function (size: TextureSize, options: DepthTextureCreationOptions): InternalTexture {
+ThinEngine.prototype._createDepthStencilTexture = function (size: TextureSize, options: DepthTextureCreationOptions, rtWrapper: WebGLRenderTargetWrapper): InternalTexture {

```

Comment:
How does this impact native ?

Reply:
It does not impact Native, as this parameter was already in the signature of the method, but not used in WebGL / WebGPU (but used in Native!).